packages/pigeon/pigeons/message.dart

```diff
@@ -20,6 +20,10 @@ import 'package:pigeon/pigeon.dart';
 /// This comment is to test enum documentation comments.
 ///
 /// This comment also tests multiple line comments.
+///
+///////////////////////////
+/// This comment also tests comments that start with '/'
+///////////////////////////
```

Comment:
I think I'd prefer a direct unit test for this instead.  I know this is better in that it is compiling the output to make sure it works, but it is also super subtle and hard to maintain.  I don't know of a good place to put it where it would get compiled and it makes sense organizationally.  We'd have to give unit tests access to the compilers probably to scale well.  Something like:
```dart
test('foo-java', () {
  String javaCode = pigeon.JavaGenerator.generate('''
// pigeon
''');
  expect(0, compile(javaCode));
});
```

That's way out of scope for this PR, just brainstorming as we consider reorganizing the tests.